.gitlab/functional_test/system_probe.yml

```diff
@@ -143,5 +150,121 @@ kitchen_test_system_probe_windows_x64:
     - export WINDOWS_DDNPM_SHASUM=$(inv release.get-release-json-value "$RELEASE_VERSION_7::WINDOWS_DDNPM_SHASUM")
     - pushd $DD_AGENT_TESTING_DIR
     - tasks/kitchen_setup.sh
+    - !reference [ .system_probe_file_diff_check ]
+  script:
+    - tasks/run-test-kitchen.sh windows-sysprobe-test $AGENT_MAJOR_VERSION
+
+kitchen_test_system_probe_linux_x64_manual:
```

Comment:
Is there a way to use a `!reference` to prevent copy/paste issues for the values that are exactly the same between the automatic and manual jobs?